about_us.html

```diff
@@ -610,82 +679,66 @@
             color: #332859;
             padding: 20px;
         }
```

Comment:
```suggestion
        }
        
        .footer-contact-info .contact-info a {
            color: #332859;
        }
```

Noticed the links weren't getting color styling on smaller screen sizes for Telephone number and email address. This will fix that.